docs/content/concepts/configuration/configured.mdx

```diff
@@ -3,7 +3,16 @@ title: Configured API | Dagster
 description: The Configured API offers a way to configure a Dagster entity at definition time.
```

Comment:
I can't comment on line 2 since it wasn't changed, but could you please add `(Legacy)` to the Frontmatter title as well?